packages/upswyng-server/src/routes/api/nextSchedule.ts

```diff
@@ -0,0 +1,100 @@
+import { ResourceSchedule, TScheduleItem } from "@upswyng/upswyng-core";
```

Comment:
I would expect uris to be snake-case (`/api/next-schedule`) also you can prob safely put this under `/api/resources`

Reply:
[update](https://github.com/CodeForBoulder/upswyng/pull/265/commits/8007b4f8556a635902c4927f1b9578ba3acd131a)